code/game/objects/items/weapons/grenades/flashbang.dm

```diff
@@ -77,44 +77,64 @@
 				ear_safety += 1
 			if(M.stats.getPerk(PERK_EAR_OF_QUICKSILVER))
 				stat_def *= 2
+	if(intensity)
+		eye_safety += 1
```

Comment:
```suggestion
		eye_safety -= 1
```
Isn't a TRUE intensity supposed to decrease the eye safety of the targets to force them to get a better flash protection to avoid the effects of the flash? With += it means an intense nade will actually do less flash (because for instance eye_safety has more chance to be bigger or equal to FLASH PROTECTION MAJOR)

Reply:
It's the opposite, in line with severity and alcohol strength, the intensity acts as a way to weaken effects.